lbz/misc.py

```diff
@@ -114,7 +115,7 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
-def copy_without_keys(data: dict, *keys) -> dict:
+def copy_without_keys(data: Union[dict, MultiDict], *keys: str) -> dict:
```

Comment:
I know you will fix it in the near future but right now, it will be more correct to write it this way:
```python
def copy_without_keys(data: MutableMapping, *keys: str) -> dict:
```